specification/paths/AddShipmentsToCollection.json

```diff
@@ -0,0 +1,57 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "shipments.manage",
+          "collections.manage"
+        ]
+      }
+    ],
+    "summary": "Add shipments to collection.",
+    "description": "This endpoint can add multiple shipments to a concept collection.",
+    "requestBody": {
+      "description": "The collection id and the shipment ids.",
+      "required": true,
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "properties": {
+              "data": {
```

Comment:
We should make `collection_id` and `shipment_ids` mandatory properties of data here. It makes no sense to call the endpoint without it.